spec/routing/bulletins_routing_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe 'routes for bulletins', type: :routing do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.